<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModalityAdmScreen</class>
 <widget class="QDialog" name="ModalityAdmScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="btn_add_modality">
     <property name="text">
      <string>Adicionar Modalidade</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_remove_modality">
     <property name="text">
      <string>Remover Modalidade</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btn_edit_modality">
     <property name="text">
      <string>Editar Modalidade</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_modality"/>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="listWidget">
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_editModality"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
